presto-orc/src/main/java/com/facebook/presto/orc/reader/ListStreamReader.java

```diff
@@ -63,19 +76,32 @@
     private int nextBatchSize;
 
     private InputStreamSource<BooleanInputStream> presentStreamSource = missingStreamSource(BooleanInputStream.class);
-    @Nullable
-    private BooleanInputStream presentStream;
 
     private InputStreamSource<LongInputStream> lengthStreamSource = missingStreamSource(LongInputStream.class);
     @Nullable
     private LongInputStream lengthStream;
 
-    private boolean rowGroupOpen;
-
     // The set of subscripts for which data needs to be
     // returned. Other positions can be initialized to null. If this
     // is null, values for all subscripts must be returned.
-    private int[] subscripts;
+    int[] subscripts;
```

Comment:
Prior comment:

---
subscripts variable is not used - remove code from setReferencedSubfields